packages/cli/uni-builder/src/shared/parseCommonConfig.ts

```diff
@@ -326,6 +327,7 @@ export async function parseCommonConfig(
     pluginGlobalVars(globalVars),
     pluginDevtool({
       disableSourceMap,
+      sourceMap,
```

Comment:
If ﻿`output.sourcemap.js` is define, I think we do not need to apply ﻿`pluginDevtool`.

Reply:
if disableSourceMap and sourceMap are set both, what behavior should we do

Reply:
`output.sourceMap`, because `output.disableSourceMap` is deprecated.

Reply:
done